cmd/build/command_test.go

```diff
@@ -347,6 +348,19 @@ func TestBuilderIsProperlyGenerated(t *testing.T) {
 			expectedError:     false,
 			isBuildV2Expected: false,
 		},
+		{
+			name: "UserFriendlyError returned without fallback.",
```

Comment:
Added test case for covering the [new logic above](https://github.com/okteto/okteto/pull/4090/files#r1429790892)